vss-accessor/tests/tests.rs

```diff
@@ -0,0 +1,272 @@
+#[cfg(test)]
+mod tests {
+	use mockito::{self, Matcher};
+	use prost::Message;
+	use vss_accessor::vss::{
+		ErrorCode, ErrorResponse, GetObjectRequest, GetObjectResponse, KeyValue, ListKeyVersionsRequest,
+		ListKeyVersionsResponse, PutObjectRequest, PutObjectResponse,
+	};
+	use vss_accessor::vss_error::VssError;
+	use vss_accessor::VssAccessor;
+
+	#[tokio::test]
+	async fn test_get() {
```

Comment:
they might look similar but they are not same.

now if i create a function which takes "endpoint", "request", "response" it would be the same thing. 
just a builder pattern vs function with same parameters.

Reply:
I am not sure if we are on same page here. 
But request creation, response creation and accessor method call is different for every case with different types. 

Just extracting create_mock_server into separate method isn't going to improve readability with its 4+ arguments when compared with a builder pattern of same parameters.

```
fn create_mock_server<M: Into<Matcher>, StrOrBytes: AsRef<[u8]>>(mock_request: M, mock_response: StrOrBytes, endpoint: Matcher, status_code: usize) -> Mock
```